hw/ip/otbn/dv/otbnsim/sim/csr.py

```diff
@@ -67,8 +71,8 @@ def write_unsigned(self, wsrs: WSRFile, idx: int, value: int) -> None:
             wsrs.MOD.write_unsigned(self._set_field(mod_n, 32, value, old))
             return
 
-        if idx == 0xfc0:
-            # RND register (writes are ignored)
+        if idx == 0xfc0 or idx == 0xfc1:
```

Comment:
Magic numbers probably want to be constants?

Reply:
I did think about it, decided to just match what was already done rather than refactor in this PR. One thing this PR introduces is the need for knowledge of the magic numbers outside of csr.py/wsr.py which makes the need for a constant instead stronger. I could refactor to keep the knowledge of what wsr/csr index is RND inside csr.py/wsr.py.

Reply:
When we do this refactor, I think the right approach is to define the constants in some YAML file. That way, we can use the same definition to support named CSRs/WSRs in the assembler.

Reply:
Sounds sensible, I'll open an issue for that refactor rather than doing it here as doing it properly with YAML defined constants, name support in assembly and feeding into doc generation will be a reasonable chunk of work.